bears/vcs/git/GitCommitBear.py

```diff
@@ -203,3 +223,80 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    body_regex: str=None,
+                    body_close_issue: bool=False,
+                    body_close_issue_url: bool=False,
+                    body_close_issue_last_line: bool=False):
+        """
+        Check for matching issue related URLs.
+
+        :param body:
+            Body of the commit message of HEAD.
+        :param body_regex:
+            If provided, matches the keywords with the regex. This takes
+            precedence over body_close_issue.
+        :param body_close_issue:
+            Whether to automatically detect host from git config or not.
```

Comment:
`body_close_issue` -> Whether to automatically detect host from git config or not

Those do not really fit together imo, maybe something like `use_git_config_remotes` or so?